cmd/storagenode/gracefulexit.go

```diff
@@ -79,22 +84,22 @@ func cmdGracefulExitInit(cmd *cobra.Command, args []string) error {
 		return errs.Wrap(err)
 	}
 
+	if len(satelliteList.GetSatellites()) < 1 {
+		fmt.Println("Can't find any non-existing satellites.")
+		return nil
+	}
+
```

Comment:
so I can't comment on line 66 since you're not actually changing it, but I was wondering if you could edit the prompt to say "Please be aware that by starting a graceful exit *from* a satellite..." instead of "on a satellite". super nitpick but I feel like being consistent the "exiting from" language could help with clarity about what's going on.